php-sdk-release/Makefile

```diff
@@ -1,16 +1,29 @@
 
 # This version should be incremented with every material change
-VERSION=1
+VERSION=2
 
 DOCKER_TAG_BASE="ldcircleci/php-sdk-release"
 
-PHPDOCUMENTOR_DOWNLOAD_URL=https://phpdoc.org/phpDocumentor.phar
+COMPOSER_VERSION=2.1.9
+COMPOSER_DOWNLOAD_URL=https://getcomposer.org/download/$(COMPOSER_VERSION)/composer.phar
+COMPOSER_SHA256=4d00b70e146c17d663ad2f9a21ebb4c9d52b021b1ac15f648b4d371c04d648ba
+COMPOSER_ARCHIVE=$(shell pwd)/downloads/composer
 
-DOWNLOADED_ARCHIVE=$(shell pwd)/downloads/phpDocumentor.phar
-LOCAL_DEPENDENCIES=$(DOWNLOADED_ARCHIVE)
+PHPDOCUMENTOR_VERSION=3.1.2
+PHPDOCUMENTOR_DOWNLOAD_URL=https://github.com/phpDocumentor/phpDocumentor/releases/download/v$(PHPDOCUMENTOR_VERSION)/phpDocumentor.phar
```

Comment:
The download URL it was using previously was for the latest version. I figure it's better to pin the version so this build is reproducible; we can update it and push a new image as needed.